infra/postman/run-e2e.sh

```
@@ -0,0 +1,215 @@
+
+LOG_DIR="$PROJECT_ROOT/infra/postman/logs"
+mkdir -p "$LOG_DIR"
+
+echo "Starting S11 (Merchant Onboarding) on port $S11_PORT..."
+java -jar "$S11_JAR" \
+  --spring.profiles.active=local,sandbox \
+  --onfido.api.base-url=http://localhost:$WIREMOCK_PORT \
+  --onfido.api.token=sandbox-token \
+  --onfido.webhook.secret=test-webhook-secret \
+  > "$LOG_DIR/s11.log" 2>&1 &
+S11_PID=$!
```

Shellcheck (0.11.0)

[warning] S11_PID appears unused. Verify use (or export if used externally).

(SC2034)